frontend/src/v4/routes/components/resources/attachResourcesDialog/attachResourcesDialog.styles.tsx

```diff
@@ -54,10 +55,13 @@ export const ResourcesListContainer = styled.div`
 	}
 `;
 
-export const ResourcesListScroller = styled.div`
+export const ResourcesListScroller = styled(ViewerScrollArea).attrs({
+	autoHeight: true,
+	autoHeightMax: isV5() ? 200 : 114,
+})`
 	margin-top: 20px;
-	overflow-y: auto;
-	width: 520px;
+    overflow: hidden;
+	width: 520px !important;
```

Comment:
I wonder why is now necessary to have the important keyword now

Reply:
I moved the styling to the parent so that it is no longer needed